onnxruntime/test/providers/cpu/nn/pool_fp16_op_test.cc

```diff
@@ -3,7 +3,7 @@
 
 #include "core/mlas/inc/mlas.h"
 
-#ifdef MLAS_F16VEC_INTRINSICS_SUPPORTED
+#if defined(MLAS_F16VEC_INTRINSICS_SUPPORTED) || defined(USE_XNNPACK)
```

Comment:
Can merge the main branch as CoreML EP also support FP16

Reply:
it has been merged